places/views.py

```diff
@@ -0,0 +1,55 @@
+from django.http import JsonResponse
+import requests, json, os
+from dotenv import load_dotenv
+
+# reading .env file
+load_dotenv()
```

Comment:
Why do you have to call load_dotenv() here again when you already called in settings.py?

Reply:
I'll take it out